go.mod

```diff
@@ -74,6 +74,203 @@ require (
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
 )
 
+require (
```

Comment:
I'm not familiar with the go mod changes in 1.17. What is this second require section for, is it different than the first?

Reply:
Paragraph 4 here has what you're after.  https://golang.org/doc/go1.17#go-command  New section to track indirect dependencies.